examples/ShowPlayer.cpp

```diff
@@ -225,17 +231,19 @@ void ShowPlayer::SendFrame(const ShowEntry &entry) const {
  * Handle the case where we reach the end of file
  */
 void ShowPlayer::HandleEndOfFile() {
+  uint64_t loop_delay = m_loop_delay;
   if (m_stop > m_playback_pos) {
     OLA_WARN << "Show file ends before the stop time (Actual length "
              << m_playback_pos << " ms)";
```

Comment:
Perhaps extend the log line to say you're pausing for x ms to catch up or something?